build/nlu-installer/dist/download.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
```

Comment:
Pretty sure these files should be gitignored!

Reply:
hmm i simply pulled master didn't write this

Reply:
https://github.com/botpress/botpress/blob/master/build/nlu-installer/.gitignore